modules/openvpn_status_log/parser.go

```diff
@@ -98,13 +101,19 @@ func parseStatusLogV2(scanner *bufio.Scanner) []clientInfo {
 	for scanner.Scan() {
 		line := scanner.Text()
 		parts := strings.Split(line, ",")
+		if len(parts) == 0 {
+			return nil
+		}
 		switch parts[0] {
 		case "HEADER":
 		case "END":
 			break
 		default:
 			switch statusType := parts[0]; statusType {
 			case "CLIENT_LIST":
+				if len(parts) < 8 {
```

Comment:
```suggestion
				if len(parts) < 9 {
```

because you access 9th element in `strconv.Atoi(parts[8])`

This is not the only place with such error, please check all if (len) checks